multiqc/modules/vep/vep.py

```diff
@@ -146,6 +146,8 @@ def parse_vep_txt(self, f):
                 continue
             key, value = line.split("\t")
             if key == "Novel / existing variants":
+                if value == "-":
+                    continue
```

Comment:
Instead of skipping this sample, can we set both `novel` and `existing` to `0`? As presumably that's what the dash means.. (should maybe double check this in the docs)